### PR TITLE
Update check ss

### PR DESCRIPTION
**Pull request by RachelXu7**
If there is element-wise add or mul and one of two variables is from a fixed-shape variable, such as input, then another variable has to be fixed too.

### CODE REVIEW

**Comment by RachelXu7** [comment]
add func that checks if there are multiple exits/output layer and fixes these output layers.

File: paddleslim/nas/ofa/ofa.py [hunk 1]
```diff
@@ -619,6 +621,22 @@ def _clear_search_space(self, *inputs, **kwargs):
...
+            ### layers in this same ss should all be fixed 
+            tmp_fixed_by_input = []
+            for ss in self._same_ss:
+                for key in fixed_by_input:
+                    if key in ss:
+                        tmp_fixed_by_input += ss
+            fixed_by_input += tmp_fixed_by_input
+
+            # tmp_output_conv = []
+            # for ss in self._same_ss:
```
Comment (ceci3) [inline paddleslim/nas/ofa/ofa.py]: 注释代码需要删掉

File: paddleslim/nas/ofa/get_sub_model.py [hunk 2]
```diff
@@ -273,11 +329,13 @@ def broadcast_search_space(same_search_space, param2key, origin_config):
...
             else:
-                if 'expand_ratio' in origin_config[pre_key]:
-                    origin_config[key] = {
-                        'expand_ratio': origin_config[pre_key]['expand_ratio']
-                    }
-                elif 'channel' in origin_config[pre_key]:
-                    origin_config[key] = {
-                        'channel': origin_config[pre_key]['channel']
-                    }
+                if pre_key in origin_config:
```
Comment (ceci3) [inline paddleslim/nas/ofa/get_sub_model.py:226]: 这里添加if分支是为了避免出现什么情况？

Comment (RachelXu7) [reply paddleslim/nas/ofa/get_sub_model.py:226]: 如果pre_key被手动fixed，或者检查出与某个固定size的var有element-wise计算，那么这个pre_key(包括其他在同一个ss中的其他keys)已经在ofa_layer中删除了，所以需要判断一下。

**Review by ceci3 — APPROVED** [review]
LGTM

**Pull request merged**